controllers/application/configmap.go

```diff
@@ -25,32 +25,46 @@ type CredentialSource struct {
 	File string `json:"file"`
 }
 
+var controllerName = "ConfigMap"
+
 func (r *ApplicationReconciler) reconcileConfigMap(ctx context.Context, application *skiperatorv1alpha1.Application) (reconcile.Result, error) {
-	controllerName := "ConfigMap"
 	r.SetControllerProgressing(ctx, application, controllerName)
 
 	// Is this an error?
-	if application.Spec.GCP == nil {
-		r.SetControllerSynced(ctx, application, controllerName)
-		return reconcile.Result{}, nil
-	}
+	if application.Spec.GCP != nil {
+		gcpIdentityConfigMap := corev1.ConfigMap{}
+
+		err := r.GetClient().Get(ctx, types.NamespacedName{Namespace: "skiperator-system", Name: "gcp-identity-config"}, &gcpIdentityConfigMap)
+		if errors.IsNotFound(err) {
+			r.GetRecorder().Eventf(
+				application,
+				corev1.EventTypeWarning, "Missing",
+				"Cannot find configmap named gcp-identity-config in namespace skiperator-system",
+			)
+		} else if err != nil {
+			r.SetControllerError(ctx, application, controllerName, err)
+			return reconcile.Result{}, err
+		}
```

Comment:
Consider creating a `getConfigMap` utility function?

Reply:
from #152 i've noticed the error handling is somewhat different between the two configmaps, so i'm not sure if it would make much sense to create a general function

Reply:
Does the error handling need to be different?

Reply:
If deciding on the more generic `GetConfigMap` approach, one could add functions for error handling like `util.ErrStopReconciling` and `util.ErrDoPanic`